pkg/monitoring/metrics/operator_metrics.go

```diff
@@ -137,7 +141,7 @@ func GetHCOMetricSystemHealthStatus(reason string) (float64, error) {
 	value := dto.Gauge.GetValue()
 
 	if err != nil {
-		return 0, err
+		return SystemHealthStatusUnknown, err
```

Comment:
so 0 is only unknown? can it be 0 also in other use cases?

Reply:
not in this metric